README.md

```diff
@@ -90,12 +90,12 @@ VCR follows the principles of [semantic versioning](https://semver.org/). The [A
 
 VCR versions 6.x are tested on the following ruby interpreters:
 
-  * MRI 2.4
-  * MRI 2.5
-  * MRI 2.6
+  * MRI 3.0
   * MRI 2.7
-
-Note that as of VCR 6, only >= 2.4 is explicitly supported.
+  * MRI 2.6
+  * MRI 2.5
+  
+Note that as of VCR 6, only >= 2.5 is explicitly supported.
```

Comment:
This comment is not precise: We have released 6.0.0, which _had_ 2.4 support.

Here is a note 

```suggestion
Note that as of after VCR 6.0.0, only >= 2.5 is explicitly supported. 6.0.0 supported >= 2.4.
```